frontend/src/container/MetricsApplication/Tabs/Overview.tsx

```diff
@@ -86,7 +84,6 @@ function Application({ getWidgetQueryBuilder }: DashboardProps): JSX.Element {
 			getWidgetQueryBuilder({
 				queryType: 1,
```

Comment:
Why is `queryType` set as a number here?

Reply:
I just keep the logic that was written before for working with queryType number because it uses in many places of the project. When we fetch the query_range, the fetch function takes queryType number and under the hood transforms it to queryType string for building correct v3 request.

Reply:
Ok, if it's easier to refactor, I would recommend updating them.